src/components/AddressInputPanel/index.tsx

```diff
@@ -102,8 +102,11 @@ export default function AddressInputPanel({
                 Recipient
               </TYPE.black>
               {address && chainId && (
-                <ExternalLink href={getEtherscanLink(chainId, name ?? address, 'address')} style={{ fontSize: '14px' }}>
-                  (View on Etherscan)
+                <ExternalLink
+                  href={getExplorerLink(chainId, name ?? address, ExplorerDataType.ADDRESS)}
+                  style={{ fontSize: '14px' }}
+                >
+                  (View on Explorer)
```

Comment:
Oopsie, Uni team missed on translate spot.
Unless it's inside `ExternalLink` component, which I didn't get to yet...

(No action to take, just a comment)